src/components/URL&Images

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+
+
+# import React from 'react'
+# import styled from 'styled-components'
+# import ProfilePicture from 'components/images/Josefine_SB.jpeg'
+# import LinkedIn from 'components/images/linkedin_icon.png'
+# import GitHub from 'components/images/github_icon.png'
+# import StackOverflow from 'components/images/stack_overflow_icon.png'
+# import HeroVideo from 'components/images/production_ID_3864185.mp4'
+
+# export const LinkedInURL = 'https://www.linkedin.com/in/josefine-engdahl-2654946a'
+# const GitHubURL = 'https://github.com/JosefineEngdahl'
+# const StackOverflowURL = 'https://stackoverflow.com/users/19477357/josefine-engdahl?tab=profile'
+# const export Other {
+
+# }
```

Comment:
**_WAIT_** what is this component? if you have time, please give me a ping and explain. I'm interested!

Reply:
hahaha, again - didn't do my clean up. I had an idea to collect all the links and images in one component so it's easier to find them, but I didn't finish it...